test/test-framework/pkg/apis/cache/v1alpha1/memcached_types.go

```diff
@@ -27,6 +27,7 @@ type MemcachedSpec struct {
 }
 type MemcachedStatus struct {
 	// Nodes are the names of the memcached pods
+	// +listType=set  //More Info: https://godoc.org/k8s.io/kube-openapi/pkg/idl
```

Comment:
Hi @joelanford, 

IMHO: This PR would not be required at all. Since all would be done in https://github.com/operator-framework/operator-sdk/pull/2129/files. I just split it to make clear the motivation as you requested. 

Could we close this one and move with the above PR? WDYT?

Reply:
Agree with adding `+k8s:openapi-gen=true` :+1:

I vote that we add that here, along with the `+listType=set` comments, and get this one merged.

Reply:
Still missing the `// +k8s:openapi-gen=true` comment directive on the `MemcachedStatus` struct.

Once that's in, this PR LGTM.